.github/workflows/ci.yml

```diff
@@ -73,36 +74,47 @@ jobs:
 
             - run: yarn encore production
 
-            - name: Bootstrap database
-              run: |
-                  bin/post-install-dev.sh
-
             - run: composer validate
               if: always()
 
-            - run: php bin/console lint:twig templates
+            - run: APP_DEBUG=1 bin/console c:w
```

Comment:
I'm not sure what you're checking by warming up the cache;

Reply:
phpstan needs the debug kernel cached version, so we need to warmup it. I've added a comment.